kintree/database/inventree_api.py

```diff
@@ -467,13 +467,16 @@ def upload_part_datasheet(datasheet_url: str, part_id: int) -> str:
     global inventree_api
 
     # Get attachment full path
-    datasheet_name = f'{os.path.basename(datasheet_url)}'
```

Comment:
This is to solve the `?` added to the datasheet name correct?

Reply:
@eeintech exactly. it also adds the benefit that you then can reverse track the datasheets to their components in the database.